lib/src/native_updater.dart

```diff
@@ -100,7 +100,7 @@ class NativeUpdater {
     try {
       AppUpdateInfo _updateInfo = await InAppUpdate.checkForUpdate();
 
-      if (_updateInfo?.updateAvailable == true) {
+      if (_updateInfo.updateAvailability == UpdateAvailability.updateAvailable) {
```

Comment:
`AppUpdateInfo` in `in_app_update 2.0.0` has no longer boolean updateAvailable attribute. it now has updateAvailability attribute which as int. and its can have 4 values 
```
class UpdateAvailability {
  UpdateAvailability._();

  static int get unknown => 0;
  static int get updateNotAvailable => 1;
  static int get updateAvailable => 2;
  static int get developerTriggeredUpdateInProgress => 3;
}
```

Reply:
For more https://pub.dev/documentation/in_app_update/latest/in_app_update/AppUpdateInfo-class.html